README.md

```diff
@@ -1,2 +1,50 @@
 # TSC
-The AdoptOpenJDK Technical Steering Comittee 
+The AdoptOpenJDK Technical Steering Comittee is the technical governing body of the AdoptOpenJDK organisation. It admits and oversees all top-level Projects in the AdoptOpenJDK organisation.
```

Comment:
Based on the following list, are you equating "top level projects" with repositories in the org?
I guess we have to be aware of the difference / overlap / relationship between the Github org and the OpenJDK adoptopenjdk group.